gm4_mountaineering/data/gm4_mountaineering/functions/climbing_effects/damage_shulkers.mcfunction

```diff
@@ -0,0 +1,10 @@
+# decrease the score of each climbing shulker
+# @s = player wearing crampons
+# at @s
+# run from climbing_effects/add_climb_effects
+
+tag @e[type=minecraft:shulker, tag=gm4_mountaineering_climb_assist] remove gm4_mountaineering_new_shulker
+scoreboard players remove @e[type=minecraft:shulker, tag=gm4_mountaineering_climb_assist] gm4_mountaineering_climbing_shulker_countdown 1
+tag @e[type=minecraft:shulker, tag=gm4_mountaineering_climb_assist, scores={gm4_mountaineering_climbing_shulker_countdown=..0}] add gm4_mountaineering_kill_shulker
+tag @e[type=minecraft:shulker, tag=gm4_mountaineering_climb_assist, distance=2.5..] add gm4_mountaineering_kill_shulker
+execute run function gm4_mountaineering:climbing_effects/kill_shulkers
```

Comment:
I'm going to leave the comment here, but this is for all of the other files as well, you don't need the `execute run` part unless you're changing the `at` or `as` first
```suggestion
function gm4_mountaineering:climbing_effects/kill_shulkers
```